GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/PublicSectorOrganisationsWithoutAPublicSectorType.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@using GenderPayGap.Database
+@model GenderPayGap.WebUI.Models.Admin.DatabaseIntegrityChecksViewModel
```

Comment:
The model here could probably be `List<Organisation>` - then you wouldn't have to do `.PublicSectorOrganisationsWithoutAPublicSectorType` loads of times

Reply:
That's neat. I reckon we can do away with the `DatabaseIntegrityChecksViewModel` completely then